spec/blueprints/child_blueprint_spec.rb

```diff
@@ -65,21 +65,35 @@
     it 'includes the correct information from the temporary dashboard case' do
       allow(Rails.application.config).to receive(:ff_live_algorithms_hours).and_return(false)
       allow(Rails.application.config).to receive(:ff_live_algorithms_full_days).and_return(false)
+      allow(Rails.application.config).to receive(:ff_live_algorithms_weekly_hours_attended).and_return(false)
       expect(JSON.parse(described_class.render(child, view: :nebraska_dashboard, filter_date: attendance_date))['hours']).to eq('11.0')
       expect(JSON.parse(described_class.render(child, view: :nebraska_dashboard, filter_date: attendance_date))['full_days']).to eq('3')
+      expect(JSON.parse(described_class.render(child, view: :nebraska_dashboard, filter_date: attendance_date))['hours_attended']).to eq('12.0')
     end
     context 'when using live algorithms' do
       it 'includes the child name and all cases' do
         allow(Rails.application.config).to receive(:ff_live_algorithms_hours).and_return(true)
+        allow(Rails.application.config).to receive(:ff_live_algorithms_full_days).and_return(true)
+        allow(Rails.application.config).to receive(:ff_live_algorithms_weekly_hours_attended).and_return(true)
+
         expect(JSON.parse(described_class.render(child, view: :nebraska_dashboard, filter_date: attendance_date))['hours']).to eq('3.0')
+        expect(JSON.parse(described_class.render(child, view: :nebraska_dashboard,
```

Comment:
@JonErvin This is also an important spec because it's combining a bunch of attendances